packages/picker/src/Picker.ts

```diff
@@ -349,14 +368,17 @@ export class PickerBase extends SizedMixin(Focusable) {
                 id="popover"
                 @click=${this.onClick}
                 @sp-overlay-closed=${this.onOverlayClosed}
-            ></sp-popover>
+            >
+                <sp-menu id="menu"></sp-menu>
+            </sp-popover>
         `;
     }
 
     protected firstUpdated(changedProperties: PropertyValues): void {
         super.firstUpdated(changedProperties);
 
-        this.optionsMenu = this.querySelector('sp-menu') as Menu;
+        this.optionsMenu = this.shadowRoot.querySelector('sp-menu') as Menu;
+        this.reparentableChildren = Array.from(this.children);
```

Comment:
I think we'll want to keep the existing usage backward-compatible for now. So here we'd look for an `sp-menu` and just reparent the children of that (and I was thinking of adding a `console.warn` DEPRECATION notice for that).